vkconfig_core/doc.cpp

```diff
@@ -442,7 +441,8 @@ bool ExportMarkdownDoc(const Layer& layer, const std::string& path) {
 
     if (!layer.settings.empty()) {
         text += "### Layer Settings Overview\n";
-        text += "|Setting|Type|Default Value|vk_layer_settings.txt Variable|Environment Variable|Platforms|\n";
+        text +=
+            "|Setting|Type|Default Value|vk_layer_settings.txt/VK_EXT_layer_settings Variable|Environment Variable|Platforms|\n";
```

Comment:
The idea is we don't want `khronos_validation.fine_grained_locking` we just want `fine_grained_locking` for the `VK_EXT_layer_settings` name

Reply:
@spencer-lunarg see the PR description for a screenshot and to get an idea what it looks like

Reply:
ok that looks nice